AuthenticatorShared/UI/Platform/Application/Extensions/String.swift

```diff
@@ -30,6 +31,12 @@ extension String {
 
     // MARK: Properties
 
+    /// Returns a SHA256-hashed version of this string as a hexadecimal string.
+    var cryptographicHash: String {
```

Comment:
@fedemkr This is the cleanup you asked for in another PR comment. 👍  Now we have one place where we do this work.

Reply:
That works for me. 👍  Updated in the latest.